src/Graphics/Glyphs/TensorGlyphBuilder.h

```diff
@@ -97,6 +110,49 @@ class SCISHARE TensorGlyphBuilder
   int nv_ = 0;
   int nu_ = 0;
 };
+
+  using MandelVector = Eigen::Matrix<double, 6, 1>;
+
+  struct DifftValues
```

Comment:
Not a fan of a big struct of public arrays, with a private magic number size. I'll look at the usage and try to suggest something.